backend/src/Edna/Library/DB/Query.hs

```diff
@@ -178,30 +178,49 @@ getMethodologyByName name = runSelectReturningOne' $ select $ do
   pure methodologies
 
 -- | Get all methodologies
-getMethodologies :: Edna [TestMethodologyRec]
-getMethodologies = runSelectReturningList' $ select $ all_ $ esTestMethodology ednaSchema
+getMethodologies :: Edna [(TestMethodologyRec, [Text])]
+getMethodologies = getMethodology' Nothing
+
+getMethodology' :: Maybe MethodologyId -> Edna [(TestMethodologyRec, [Text])]
```

Comment:
Why not to return `[WithId 'MethodologyId MethodologyResp]`

Reply:
I think that using API types in Service modules is something undesirable. I don't want to prohibit it because sometimes it's convenient, but in general I think that conversion between DB types and API types should be the goal of Service layer.